routing/blinding.go

```diff
@@ -1,17 +1,23 @@
 package routing
 
 import (
+	"bytes"
```

Comment:
nit: I'm not 100% sure we want to mention different designs in the commit message as that could be confusing, but open to it

Reply:
removed the different design part.